*** tests/A-014-014-M-ok.out	Fri Oct  6 12:23:45 2023
--- tests/A-014-014-M-ok.outhyp	Thu Oct 12 03:52:56 2023
***************
*** 14,23 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|5
! 1;2|49
! 1;3|5=1;1
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 14,20 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 26,31 ****
--- 23,29 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
